### PR TITLE
Radiobrowser

### PR DESCRIPTION
**Pull request by elpatron68**
#49 
Support for http://www.radio-browser.info API

Added two new commands:

- `!rbquery <station name>`: Search for a radio station. Search results will be posted to channel as html table of "id" and "station name"
- `rbplay <station id>`: Play a specific radio station

### CODE REVIEW

**Comment by elpatron68** [comment]
Added new commands to README.md

**Comment by azlux** [comment]
I see your IDE follow PEP8 guideline (the line length limited to 79).
Not a bad thing, just an habit I have on large screen. It's create lot of changes :D 

You code look good. Shouldn't be better to use a new folder to external resources ? You have create 3 files. The root folder become messy.

Can you remove your backup (` mumbleBot.py.bak`) ?

**Comment by elpatron68** [comment]
Sorry, my editor had problems with your tabs, tried to fix it (for me).

I will try to move the new files to ./lib - this seems to be the "pythonic" way? Never tried.

How about logging. II info level okay or should we use debug?

**Comment by azlux** [comment]
You can use space instead of tab, no problem (it's the recommendation)

Use info for
- "Found url: ' + url"
- "No playable url found"

The others are debug informations.

Using `lib` folder is good too.

**Comment by elpatron68** [comment]
This should work so far. At least on my Mumble machine. Please test.

File: mumbleBot.py [hunk 1]
```diff
@@ -398,6 +398,7 @@ def message_received(self, text):
 			                     'url': url,
 			                     'user': user}
                         var.playlist.append(music)
+                        self.send_msg('Now playing ' + radiobrowser.getstationname_byid(parameter), text)
```
Comment (azlux) [inline mumbleBot.py]: Now playing should be done late by the bot.
https://github.com/azlux/botamusique/blob/master/mumbleBot.py#L518

Comment (elpatron68) [reply mumbleBot.py]: I´m thinking of replacing the bot´s message against a table with information about the radio station from radio-browser, which is more accurate and contains more information.

Any idea how to disable the bot message when playing a rb station?

_Bot message_
```
[22:43:47] (Kanal) botamusique: Now playing http://regiocast.hoerradar.de/nora-103-mp3-mq?sABC=5q3p46ps%230%23o000r827s931po57p4q48q3377r73615%23ubzrcntr&amsparams=playerid:homepage;skey:1564231375
URL : http://regiocast.hoerradar.de/nora-103-mp3-mq?sABC=5q3p46ps%230%23o000r827s931po57p4q48q3377r73615%23ubzrcntr&amsparams=playerid:homepage;skey:1564231375
```
_RB message_

| ID | Station Name | Genre | Codec/Bitrate | Country | Homepage |
| -- | -- | -- | -- | -- | -- |
| 96748 | NORA Oldies | 60s,70s | MP3/128 | Germany | http://www.norawebstreams.de/ |

Comment (elpatron68) [reply mumbleBot.py]: Shall I remove the message? Otherwise: No more commits from me concerning this pr.

**Comment by elpatron68** [comment]
I have modified the query answer (will be reduced twice it if is too long, limit is 5000 chars).

I think, now it´s done and full featured. Please just tell me if I shall remove the "Playing..." message, which is atm somehow redundant. Thanks and I wish you a nice weekend!

**Comment by elpatron68** [comment]
I used Python 3's f-Strings which would require Python 3.6 or higher. As this could be a problem for some older systems, I will change this to %-formatting to support older Python versions in the upcoming commit.

**Pull request merged**